.github/workflows/build-git-installers.yml

```diff
@@ -187,24 +187,23 @@ jobs:
         run: |
           set -x
 
-          eval /usr/src/build-extra/please.sh make_installers_from_mingw_w64_git --version=${{ needs.prereqs.outputs.tag_version }} -o artifacts --${{matrix.artifact.name}} --pkg=pkg-x86_64/mingw-w64-x86_64-git-[0-9]*.tar.xz --pkg=pkg-x86_64/mingw-w64-x86_64-git-doc-html-[0-9]*.tar.xz &&
+          # Copy the PDB archive to the directory where `--include-pdbs` expects it
+          b="/usr/src/build-extra" &&
```

Comment:
The double quotes are not needed here because there is no special character nor space in the value. Maybe lose the quotes?

Reply:
Removed quotes & amended this comment to `fixup!` the Windows installer commit from the original pull request.